src/devices/Card/CreditCard/Source.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Card.CreditCardProcessing
+{
+    /// <summary>
+    /// Th esourvce of a Tag
```

Comment:
spac eoff by 1

Reply:
and source not written correctly :-) fixed.